Documentation/En/Index.xyl

```diff
@@ -26,28 +26,28 @@
   abbreviated SUT, and is composed of two parts:</p>
   <ol>
     <li><strong>Test data</strong> to execute the SUT, and</li>
-    <li>An <strong>oracle</strong>, establishing the <strong>test
+    <li>an <strong>oracle</strong>, establishing the <strong>test
```

Comment:
In English, we put the first word of every item of a list with an upper-case. This is not the case in other languages like French or Italian, or even Arabic though.